lib/ansible/modules/net_tools/basics/uri.py

```diff
@@ -53,8 +53,7 @@
     version_added: "2.0"
   method:
     description:
-      - The HTTP method of the request or response. It MUST be uppercase.
-    choices: [ CONNECT, DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT, REFRESH, TRACE ]
+      - The HTTP method of the request or response.
```

Comment:
i would note 'since 2.8 we don't restrict this at the module level anymore but it still must be a valid method accepted by the service handling the request'

Reply:
Ok. Will add!

Reply:
But would like to backport to v2.7 and actually also v2.6...